test/logger.js

```diff
@@ -45,6 +45,23 @@ test('only write if reset is supported by reporter', function (t) {
 	t.end();
 });
 
+test('only call clear if supported by reporter', function (t) {
+	var tapReporter = tap();
+	var logger = new Logger(tapReporter);
+	tapReporter.clear = undefined;
+	logger.clear();
+	t.end();
+});
+
+test('only write if clear is supported by reporter', function (t) {
+	var tapReporter = tap();
+	var logger = new Logger(tapReporter);
+	tapReporter.clear = undefined;
+	logger.write = t.fail;
```

Comment:
[Looks](https://github.com/tapjs/node-tap/blob/d5e11d0f660ac0a38b486adfb505b28a02e51fd3/lib/test.js#L118) [like it](https://github.com/tapjs/node-tap/blob/d5e11d0f660ac0a38b486adfb505b28a02e51fd3/lib/test.js#L1363)